src/main/ngapp/accio/src/app/editor/editor.component.ts

```diff
@@ -457,11 +493,11 @@ export class EditorComponent implements OnInit {
    *    of gallery images. If the user clicked on a specific mask, returns the next mask.
    *  @param previous signifies whether the user has selected the previous image button.
    */
-  switchImage(previous: boolean) {
+  switchImage(direction: SwitchImage) {
     // If user clicks on an image's mask, then newImage will loop through all the image's masks.
     if (this.maskIndex == 0 || this.maskIndex) {
       let maskObject = this.imageArray[this.index]['masks'];
-      if (previous) {
+      if (direction == SwitchImage.PREVIOUS) {
```

Comment:
Nit. Should use === vs == in .js, as == can have unintended behavior

Reply:
Weird because when I tried to do this earlier in the project I am pretty sure it gave me an error but it;s good now. Thanks for pointing it out.